biojava-structure/src/main/java/org/biojava/nbio/structure/cluster/SubunitCluster.java

```diff
@@ -71,11 +74,10 @@
  */
 public class SubunitCluster {
 
-	private static final Logger logger = LoggerFactory
-			.getLogger(SubunitCluster.class);
+	private static final Logger logger = LoggerFactory.getLogger(SubunitCluster.class);
 
-	private List<Subunit> subunits = new ArrayList<Subunit>();
-	private List<List<Integer>> subunitEQR = new ArrayList<List<Integer>>();
+	private List<Subunit> subunits = new ArrayList<>();
```

Comment:
Why did you remove the Class of elements in the List when it is defined? Has something changed in Java8+?

Reply:
Since Java 7 it is redundant to use the types in the variable initialization. They `<>` is called "diamond". See "The diamond" section here: https://docs.oracle.com/javase/tutorial/java/generics/types.html

I think it is good style to remove them, since they are redundant. It helps readability.